src/utils/http.js

```diff
@@ -33,12 +46,10 @@ function json(response) {
   return response.json();
 }
 
-export default function http(method, url, data , headers = {}) {
+export function http(method, url, data, headers = {}) {
```

Comment:
all in for this, but JS is shouting at me for using `delete`... want to alias it to destroy instead? I don't like it :(